### DISCUSSION TITLE
Multiselect DataGrid row disable

### DISCUSSION THREAD

**Discussion by afreidz** [microsoft/fluentui#29344] [Q&A]
is there a way to disable a particular row in a multi-select DataGrid? Or more specifically, I will have a mixture of items that are selectable by themselves, and items that will have children that are selectable (the parent will not be).  i think i will be able to handle rendering the children using `noNativeElements` and adding my own UI to show "children" ... but unsure how to make the parent record select disabled.

**Comment by JHaoX** [comment]
DataGridRow has a property `selectionCell` to control the selection cell. 

```
<DataGridRow
  selectionCell = {{
    checkboxIndicator: {disabled: [disable logic]}
  }}
>
```

**Comment by lennaylennay2** [comment]
While this does work for the react state triggers the row is still visablly selecvtable even when disabled.
